test_runner/src/main/kotlin/ftl/filter/TestFilters.kt

```diff
@@ -114,8 +114,18 @@ object TestFilters {
         }
     }
 
-    private fun withSize(args: List<String>): TestFilter =
-        withAnnotation(args.map { Size.from(it).annotation })
+    private fun withSize(args: List<String>): TestFilter = args.map { Size.from(it).annotationName }.let { annotationNames ->
+        TestFilter(
+            describe = "withSize (${annotationNames.joinToString(", ")})",
+            shouldRun = { testMethod ->
+                // Ensure that all annotation with a name matching this size are detected, like
+                // https://developer.android.com/reference/android/support/test/filters/LargeTest or
+                // https://developer.android.com/reference/androidx/test/filters/LargeTest
+                testMethod.annotationNames.any { it.split(".").last() in annotationNames }
+            },
+            isAnnotation = true
```

Comment:
The original implementation was reusing  withAnnotation so `isAnnotation` was true + size is annotation based I figure. Please let me know if this make sense or this was supposed to only match (not)withAnnotation instructions.